source/game.programme.programmedata.bmx

```diff
@@ -477,11 +479,9 @@ Type TProgrammeDataCollection Extends TGameObjectCollection
 	End Function
 
 	'=== EVENT HANDLERS ===
-	Rem
 	Function onSetLanguage:int( triggerEvent:TEventBase )
 		GetInstance().RemoveReplacedPlaceholderCaches()
```

Comment:
Hier bin ich mir jetzt nicht sicher, ob es dann beim Sprachwechsel auch zur Neuauswuerflung von "Variablen" kommen kann. Bzw. Neuauswuerflung von "var1|var2|var3".

Reply:
In ProgrammeData sollte es doch keine Alternativen mehr geben.
Wenn alles richtig ist, dürften in ProgrammaData nur noch Personen-Referenzen etc. drin sein.